helpers/publicarMovimientoDetalle.go

```diff
@@ -47,7 +48,7 @@ func PublicarMovimientosDetalle(idMovProcExterno int) (movimientosDetalleRespues
 		return []models.MovimientoDetalle{}, formatErr
 	}
 
-	// logs.Debug("LISTA DE RUBROS: ", listaRubros)
+	logs.Debug("LISTA DE RUBROS: ", listaRubros)
```

Comment:
Dejar comentado o eliminar
```suggestion
	// logs.Debug("LISTA DE RUBROS: ", listaRubros)
```